### DISCUSSION TITLE
Controlling Jandy WaterColors LED lights using Nixie/REM?

### DISCUSSION THREAD

**Discussion by Grauwulf75** [tagyoureit/nodejs-poolController#815] [Nixie]
Looking for some help integrating the pool lights that we have in to NjsPC. They are Jandy WaterColor LED's with 14 different color modes, switched by switching the power off/on with a 3 second interval. The lights can be reset to the beginning of the sequence by turning the lights off and then back on after a 5 sec delay. If you leave them off for more than 7 seconds, it turns back on at the last selected mode.

I don't know a ton about coding, but could probably muddle my way through it if someone could point me in the right direction. I'm running NjsPC/Dashboard/REM on a RaspberryPi 3b+ using an 8-relay bank to control my devices.

Thanks!

**Comment by rstrouse** [comment]
If you look at the definitions starting on line 175 of NixieBoard.ts you will find the color definitions.  Right now we have colors for intellibrite, magicstream, colorlogic, and pooltone.  Then in controller/nixie/circuits/Circuit.ts you will find all the code that does the color timings.

**Reply by Grauwulf75** [reply]
I've taken a look at those two files and it's above my head,  unfortunately. Any chance of making it a feature request?

**Reply by tagyoureit** [reply]
Pull the changes.  I just added the code for this.  Let us know if it works.
